### DISCUSSION TITLE
Support adjustable heights and widths for embedded content

### DISCUSSION THREAD

**Discussion by oliver-mower** [GitbookIO/community#452] [Feature Requests]
Right now, when you embed a url, the height of the embed is pre-determined and can't be adjusted. This is challenging as the content rendered in those embeds varies and should be adjustable by the end user. 

With codepen for example, the height is already very narrow to begin with:
<img width="778" alt="Screenshot 2024-01-12 at 4 06 20 PM" src="https://github.com/GitbookIO/community/assets/110262783/616815b3-4166-4629-8262-54fbdb3d61ff">

 It would be great if the user could drag and adjust the height and width like they can in other documentation tooling such as notion:

![Jan-12-2024 16-08-57](https://github.com/GitbookIO/community/assets/110262783/b5d25a36-92f0-46d7-a055-ec128f67cb3b)

**Comment by aikoii** [comment]
👋 @oliver-mower! Thanks for sharing your feedback.

We have a similar suggestion here https://github.com/orgs/GitbookIO/discussions/426, I encourage you to vote for it. 

We don't have any immediate plans to support adjustable heights and widths for embedded content, but hoping to offer a way in future 🙂.

(I will be closing this in favor of the first request)
